wdl-format/tests/format.rs

```diff
@@ -128,11 +127,21 @@ fn run_test(test: &Path) -> Result<(), anyhow::Error> {
     let formatted_path = path.with_extension("formatted.wdl");
     let source = std::fs::read_to_string(&path).context("reading source file")?;
 
-    let formatted = format(&source, path.as_path())?;
+    let mut builder = Builder::default();
+    let t = test.file_stem().unwrap().to_string_lossy();
+    if t == "input_sorting" {
+        builder = builder.sort_inputs(true);
+    }
+    if t == "parameter_meta_sorting" {
+        builder = builder.sort_parameter_meta(true);
+    }
+    let formatter = Formatter::new(builder.build());
+
+    let formatted = format(&source, path.as_path(), &formatter)?;
```

Comment:
> sprocket format and sprocket lint are separate use cases. Someone using the format command might not see any of the lint diagnostics.

> I'm still undecided on whether input/parameter_meta sorting should be "opt in" or "opt out" behavior, but I think if you get this implemented we can play around with it on some real WDL repos to help us decide!
\- @a-frantz 

This can be simplified if `sort_inputs` and `sort_parameter` are combined. But i'm not sure if @a-frantz meant single or combined configs above.